lib/test_queue/runner/minitest4.rb

```diff
@@ -66,4 +69,20 @@ def run_worker(iterator)
       end
     end
   end
+
+  class TestFramework
+    class MiniTest < TestFramework
+      def all_suite_paths
+        ARGV
+      end
+
+      def suites_from_path(path)
+        ::MiniTest::Unit::TestCase.reset
+        require File.absolute_path(path)
```

Comment:
> What happens if we put a directory in `ARGV`? I think this would explode? Would minitest?

test-queue will explode (with a `Discovering suites failed` error), just like it would before this PR. minitest doesn't do any loading of test files itself; that's entirely up to the caller.

The intent is that this method is only ever called with a path to a file. I'll rename to `suites_from_file`.